src/components/calcite-dropdown-item/calcite-dropdown-item.scss

```diff
@@ -6,14 +6,17 @@
 // ie11
 :host-context([scale="s"]) {
   --calcite-dropdown-item-padding: #{$baseline/5 $baseline/1.5 $baseline/5 $baseline * 1.5};
+  font-size: theme("fontSize.-2");
 }
 
 :host-context([scale="m"]) {
   --calcite-dropdown-item-padding: #{$baseline/3 $baseline/1.5 $baseline/3 $baseline * 1.5};
+  font-size: theme("fontSize.-1");
 }
 
 :host-context([scale="l"]) {
   --calcite-dropdown-item-padding: #{$baseline/2 $baseline/1.5 $baseline/2 $baseline * 1.5};
+  font-size: theme("fontSize.1");
```

Comment:
We should also update the calcite-dropdown-group group-title text to match these changes, probably to the same values.